config/index.js

```diff
@@ -0,0 +1,20 @@
+const generateTestConfig = require('./generateTestConfig')
+const generateProdConfig = require('./generateProdConfig')
+const generateDevConfig = require('./generateDevConfig')
+
+require('dotenv').config()
+
+const getConfig = (env) => {
+  switch (env) {
+    case 'test':
+      return generateTestConfig()
+    case 'production':
+      return generateProdConfig()
+    case 'development':
+      return generateDevConfig()
+    default:
+      throw new Error('Set NODE_ENV to either production | development | test')
+  }
+}
```

Comment:
At the risk of 'testing implementation' over functionality, this could have been tested 🤷 
I concede its simple enough that we don't gain much from the test though :-)